<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TcpServer</class>
 <widget class="QDialog" name="TcpServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>发送端</string>
  </property>
  <widget class="QLabel" name="serverStatusLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>请选择要发送的文件！</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="serverOpenButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </widget>
  <widget class="QPushButton" name="serverSendButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="serverCloseButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>120</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
